<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer_widget</class>
 <widget class="QWidget" name="Viewer_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>View &amp;&amp; Edit</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" rowstretch="0" columnstretch="0">
       <item row="0" column="0">
        <widget class="Viewer" name="graphicsView" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Mask</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Origin:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="Viewer" name="graphicsView_origin" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Result:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGraphicsView" name="graphicsView_Result">
            <property name="backgroundBrush">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>200</red>
               <green>200</green>
               <blue>200</blue>
              </color>
             </brush>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Mask:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="Viewer" name="mask_viewer" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="MaskSettings" name="mask_settings" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaskSettings</class>
   <extends>QWidget</extends>
   <header>../WTF_Viewer/viewer/masksettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>../WTF_Viewer/viewer/viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>graphicsView_Result</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
